nvflare/app_opt/xgboost/histogram_based_v2/fed_executor.py

```diff
@@ -50,12 +51,15 @@ def __init__(
         self.in_process = in_process
 
     def get_adaptor(self, fl_ctx: FLContext):
+
+        if not self.client_handler_id:
```

Comment:
See my comment about this in Controller.

Reply:
Removed.